modules/persistence/tests/metadata/utils/convertSlugToUrl.test.ts

```diff
@@ -1,9 +1,15 @@
 import { convertSlugToUrl } from '../../../src/services/metadata/ToC/utils/convertSlugToUrl';
 
 describe('convertSlugToUrl', () => {
-  it("accepts a project's prefix, slug, and base url, and returns an absolute url for href usage", () => {
-    expect(convertSlugToUrl('get-started', 'docs/atlas/cli', 'www.mongodb.com')).toEqual(
-      'www.mongodb.com/docs/atlas/cli/get-started'
+  it("accepts a project's prefix, slug, alias and base url, and returns an absolute url for href usage", () => {
```

Comment:
minor nitpick. would change the args to `atlas-cli-getting-started` and `upcoming` 
![image](https://user-images.githubusercontent.com/13054820/219500237-4afcd7e8-aa65-456c-ab95-24be160e1919.png)